Commands.md

```diff
@@ -851,6 +852,16 @@ Deleted feature/dashboard (was 923befa).
 ...
 ```
 
+## git delete-squashed-branches
+
+Deletes branches that have been "squashed-merged" into a specified branch
+
+```bash
+$ git delete-squashed-branches main
```

Comment:
Should also update the doc

Reply:
How does this sound to you?

Reply:
Should we also need to update the output of `git delete-squashed-branches main`?
Would be better if we use `$ (branch) git ...` to indicate the branch change.

Reply:
Sure! Is this a new convention or are there other commands that do this too?

Reply:
How does this sound to you?

~~~markdown
```bash
$ (feature-branch) git delete-squashed-branches main
Deleted branch dependabot/bundler/kramdown-2.3.1 (was 1d3fb00).
Deleted branch dependabot/bundler/rexml-3.2.5 (was a7e4052).
$ (main) git ...
```
~~~

Reply:
Alrighty! Amended to my last commit